### DISCUSSION TITLE
View - Ground Rules

### DISCUSSION THREAD

**Discussion by jin-Pro** [githru/githru-vscode-ext#9] [View]
- 유지보수를 잘 하기 위해 공통으로 적용할 항목들에 대해 협의합니다.
  - eslint,prettier rule sets
  - test 전략 (테스트 범위, coverage, ...)
  - coding conventions
    - directory structure
    - file naming
    - variable naming
  - ...

- 머지 기준
  - 테스트 코드 통과 및 커버리지 통과
  - 리뷰어
  - 승인 인원수
  - 머지 방법

- 이슈 / 위키 관리

- 회의

수정 및 보완할 내용 적극적으로 말씀해주세요!

**Comment by ytaek** [comment]
제가 내일 아마 참여하겠지만, 일단 플젝 전반적인 일관성을 위해 아래는 고정했으면 하네요.
- 승인 2명
- eslint/prettier부분은 tsx부분 이외에는 engine과 동일하게 맞추기
